source/scenario/scenario1a.vwf.yaml

```diff
@@ -103,15 +103,15 @@ children:
             - [ 13, 15, -15 ]
           - setObjective:
             - "Click the PROGRAM button to open your control processor."
-          - setHUDProperty:
-            - camera_topDown
```

Comment:
Can we get rid of the setHUDProperty action?  Any other actions we can get rid of/simplify now?